internal/services/order/stocks_feed.go

```diff
@@ -0,0 +1,24 @@
+package order
+
+import (
+	"Tradeasy/config"
+	"Tradeasy/internal/model"
+	"Tradeasy/internal/services/stock_exchange"
+)
+
+func UpdateStocksFeed(res []stock_exchange.StockDetails) (err error){
+	for _,stock:=range res{
+		newStock:=model.StocksFeed{
+			StockName: stock.StockName,
+			LTP:       stock.LTP,
+			Open:      stock.Open,
+			High:      stock.High,
+			Low:       stock.Low,
+			TradedAt:  stock.UpdatedAt,
+		}
+		if err=config.DB.Table("stocks_feed").Create(&newStock).Error;err!=nil{
```

Comment:
are we not updating stock feed even if one fails?

Reply:
Added continue instead of return